testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -70,4 +70,48 @@ TestServer_clientID_booted : UnitTest {
 		s.quit;
 		s.remove;
 	}
+
+	test_recoverRemoteReLogin {
+		var options, homeServer, remote, dt = 0.1, timeout = 5;
+
+		options = ServerOptions.new;
+		options.maxLogins = 4;
+		options.sampleRate = 48000;
+
+		homeServer = Server(\pseudoHome, options: options);
+
+		homeServer.clientID = 3;
+		this.bootServer(homeServer);
+
+		this.assert(homeServer.clientID == 3,
+			"homeServer gets requested clientID back from server process."
+		);
+
+		// make s play dead now, but leave the process running
+		homeServer.statusWatcher.stopStatusWatcher.stopAliveThread.serverRunning_(false);
+		// homeServer.serverRunning.postln; // client thinks it is dead
+
+		// now login again from different server object, but same client address
+		// -> this client netaddr is already registered, and should say so!
+		// -> so the response should go thru prHandleNotifyFailString
+
+		remote = Server.remote(\remTest, homeServer.addr, homeServer.options);
```

Comment:
flow of logic would make more sense to me if this server were created right after the homeServer; does this constructor call depend on preceding code in a way I'm not seeing?